x-pack/plugins/endpoint/common/types.ts

```diff
@@ -311,8 +311,8 @@ export interface EndpointEvent {
     version: string;
   };
   event: {
-    category: string;
-    type: string;
+    category: string | string[];
+    type: string | string[];
```

Comment:
when would category and type be string arrays?

Reply:
I'm not sure exactly when this is the case, I believe some events can have more than one of each in the case where it was hard to bucket into exactly one existing ECS value. @jonathan-buttner could probably tell you in more detail, this comes from https://github.com/kqualters-elastic/kibana/pull/1